frontend/src/map/map.jsx

```diff
@@ -158,7 +160,7 @@ const map = new Map({
     scenarioLayerGroup,
   ],
   view: new View({
-    center: [-10984368.72, 3427876.58], // W. San Antonio, EPSG:3857
+    center: [-10956248.2268683, 3427877.42426285], // E. San Antonio, EPSG:3857
```

Comment:
Woah, this is going to throw me!

Reply:
Hmm, now that the map zoom's to the selected study area, and session's are saved in the browser, with a study area pre-selected if one exists, we could probably change this to just center on the city at a smaller zoom. And in practice that would only happen for a fresh session with no study areas.